packages/ddp-client/common/livedata_connection.js

```diff
@@ -256,7 +267,15 @@ export class Connection {
       });
     }
 
-    const onDisconnect = () => {
+    const onDisconnect = (e) => {
+      if (e?.code) {
+        // We don't want to retry the outstanding methods for the following error cases
+        self.shouldRetryMethods = ![WEBSOCKET_ERROR_CODES.too_large, WEBSOCKET_ERROR_CODES.extension_error].includes(e?.code);
+
+        self.lastDisconnectError = new Meteor.Error(e.code, e.reason);
+        console.log(`DDP Connection closed because of: [${e.code}] ${e.reason}`);
```

Comment:
Why `connection` is upper-case here?

Reply:
Also, shouldn't we actually use `Meteor._debug` here? (for internal meteor consistency)